src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1354,7 +1354,9 @@ internal void AddPredefinedPropertyToSymbolTable(AggregateSymbol type, Name prop
         private void AddPropertyToSymbolTable(PropertyInfo property, AggregateSymbol aggregate)
         {
             Name name;
-            bool isIndexer = property.GetIndexParameters() != null && property.GetIndexParameters().Length != 0;
+            bool isIndexer = property.GetIndexParameters().Length != 0
```

Comment:
Any reason to use predicateless-Any() on an array? The perf difference isn't much, but it's there, and `Length != 0` is common with arrays elsewhere in corefx.